md/fragments.md

```diff
@@ -26,7 +26,7 @@ You can create a fragment from scratch in the UI Designer home page.
 You can also convert a plain container used in a page, a form, a layout or a fragment into a fragment:
 
 1. Select the container in the page editor.
-2. Click the link **Save as fragment** in the properties panel.
+2. Click the link **Save as fragment** in menu option close to the fragment name in the properties panel.
```

Comment:
"2. From the properties panel header, click the "Widget actions" menu and select "Save as fragment" item"